sklearn/metrics/_ranking.py

```diff
@@ -923,6 +933,15 @@ def precision_recall_curve(
 
         .. versionadded:: 1.3
 
+    probas_pred : array-like of shape (n_samples,)
+        Target scores, can either be probability estimates of the positive
+        class, or non-thresholded measure of decisions (as returned by
+        `decision_function` on some classifiers). By default None.
```

Comment:
The `None` semantics is an implementation detail, so we can deprecate  `probas_pred` nicely. I do not think we need to state it for users:

```suggestion
        `decision_function` on some classifiers).
```